### PR TITLE
Add authorship

### PR DESCRIPTION
**Pull request by hcorson-dosch-usgs**
Adds in and populates the authorship component from Vue scratch:
image

I also ran lint (those are the changes in `DroughtThresholds.vue`

And I took out some classes that were either unused or setting a different font than the page font within `References.vue`

### CODE REVIEW

File: src/assets/text/authors.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+export default {
+    // do not delete section. delete individuals as needed. modify content as needed
+    // currently only 'fullName', 'firstName', 'initials', 'profile_link', and 'contribution' are used
+    primaryAuthors: [
+      {
+        firstName: 'Althea',
+        lastName: 'Archer',
+        fullName: 'Althea Archer',
```
Comment (aaarcher-usgs) [inline src/assets/text/authors.js]:
```suggestion
        fullName: 'Althea A. Archer',
```

File: src/assets/text/authors.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        initials: 'AA1',
```
Comment (aaarcher-usgs) [inline src/assets/text/authors.js]:
```suggestion
        initials: 'AAA',
```

**Review by aaarcher-usgs — APPROVED** [review]
Small changes. I like to use my middle initial when I can. Triple A!

**Comment by aaarcher-usgs** [comment]
I also ran lint in the pull request with the aria additions. I had one error after running lint that I had to fix, but I don't see it in your code so we're ok!

File: src/assets/text/authors.js (hunk 2 above)
Comment (hcorson-dosch-usgs) [reply src/assets/text/authors.js]: thanks for catching those holdovers from the template!

**Pull request merged**